### PR TITLE
common&axisSigFix-tests

### PR DESCRIPTION
**Pull request by joyrostie**

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-235.gearboxjs.preview.cogniteapp.com

File: src/mocks/tests/common.test.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { generateNumber, sleep } from '../common';
+
+configure({ adapter: new Adapter() });
+
+describe('generateNumber', () => {
```
Comment (mrtraser) [inline src/mocks/tests/common.test.ts]: Could you redo this (related to number of digits) with approach that @follan uses there:
https://github.com/cognitedata/gearbox.js/blob/master/src/components/TimeseriesChart/dataLoader.test.tsx#L84

Comment (joyrostie) [reply src/mocks/tests/common.test.ts]: done :-)

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-235.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-235.gearboxjs.preview.cogniteapp.com

**Pull request merged**